### PR TITLE
[Docs] translate /demo/wasi to zh

### LINKED ISSUE #1074: [Docs] Translate the demo/wasi.md to zh
### The MD file link you want to translate (must be in English) 
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/kubernetes/demo/wasi.md

### Target language
zh: https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/zh/src/kubernetes/demo/wasi.md

### PR DESCRIPTION
**Pull request by wangyuan249**
Signed-off-by: wangyuan249 <249401644@qq.com>

fix #1074

### CODE REVIEW

File: docs/book/zh/src/kubernetes/demo/wasi.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# 简单的 WASI 例子
+# 一个简单的 WASI 例子
+
+在这片文章中，我会向你展示如何去构建一个 WebAssemblely 应用的容器镜像。然后可以通过 Kubernetes 生态系统工具（例如 CRI-O、Docker、crun 和 Kubernetes）来启动和管理它。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo/wasi.md:3]: 片 --> 篇

File: docs/book/zh/src/kubernetes/demo/wasi.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+
+## 先决条件
+
+> 如果你只是想要一个 wasm 字节码文件作为容器镜像进行测试，你可以跳过构建步骤，只需[在此处下载 wasm 文件](https://github.com/second-state/wasm-learning/blob/master/cli/wasi/wasi_example_main.wasm)。
+
+如果你还没有这样做，请按照这些简单的说明来[安装 Rust](https://www.rust-lang.org/tools/install)。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo/wasi.md:9]: 这里改成，首先，请按照这些简单的说明来[安装 Rust](https://www.rust-lang.org/tools/install)。

**Review by alabulei1 — COMMENTED** [review]
Great work! there are some typos that need to be taken care of.

**Comment by wangyuan249** [comment]
please check again, thanks @alabulei1

**Pull request merged**